### PR TITLE
Add elevation rendering to tourism=wilderness_hut

### LINKED ISSUE #4566: Render elevation for tourism=wilderness_hut
### Expected behavior
As with `tourism=alpine_hut` and `amenity=shelter`, `tourism=wilderness_hut` features should have `ele` rendered

### Actual behavior
Only the name is rendered for `tourism=wilderness_hut`, not the elevation

There are about 2000 features (16% of all tourism=wilderness_hut) which would be affected by this change.

### PR DESCRIPTION
**Pull request by wyskoj**
Fixes #4566 

Changes proposed in this pull request:
- Adds elevation rendering for `tourism = wilderness_hut`

Test rendering with links to the example places:
https://www.openstreetmap.org/way/985409968#map=19/48.49545/-121.20517

**Before**
download (4)

**After**
download (3)

### CODE REVIEW

**Review by sommerluk — APPROVED** [review]
Tested. Works fine.

File: project.mml [hunk 1]
```diff
@@ -1475,6 +1475,7 @@ Layer:
                       WHEN (tags ? 'ele') AND tags->'ele' ~ '^-?\d{1,4}(\.\d+)?$'
                         AND ("natural" IN ('peak', 'volcano', 'saddle')
                           OR tourism = 'alpine_hut' OR (tourism = 'information' AND tags->'information' = 'guidepost')
+                          OR tourism = 'wilderness_hut'
```
Comment (pnorman) [inline project.mml:1478]: Combine this into the alpine_hut condition by modifying it to `tourism IN ('alpine_hut', 'wilderness_hut')`

**Review by pnorman — CHANGES_REQUESTED** [review]
Looks okay cartographically, just a minor code formatting change.

File: project.mml (hunk 1 above)
Comment (wyskoj) [reply project.mml]: Thanks, fixed.

**Review by pnorman — APPROVED** [review]

**Pull request merged**